src/components/WorkOrder/TasksAndSors/TasksAndSorsRow.js

```diff
@@ -1,28 +1,26 @@
 import PropTypes from 'prop-types'
 import { formatDateTime } from '../../../utils/time'
+import { TR, TD } from '../../Layout/Table'
 
 const tasksAndSorsRow = ({
   code,
   description,
   dateAdded,
+  isOriginal,
+  originalQuantity,
   quantity,
   cost,
   status,
   index,
 }) => (
-  <tr
-    data-row-id={index}
-    className="govuk-table__row govuk-table__row--clickable govuk-body-s"
-  >
-    <td className="govuk-table__cell">{code}</td>
-    <td className="govuk-table__cell">{description}</td>
-    <td className="govuk-table__cell">
-      {dateAdded ? formatDateTime(dateAdded) : '—'}
-    </td>
-    <td className="govuk-table__cell">{quantity}</td>
-    <td className="govuk-table__cell">{cost}</td>
-    <td className="govuk-table__cell">{status}</td>
-  </tr>
+  <TR index={index} className="govuk-body-s">
+    <TD>{code}</TD>
+    <TD>{description}</TD>
+    <TD>{dateAdded ? formatDateTime(dateAdded) : '—'}</TD>
+    <TD>{isOriginal ? originalQuantity : quantity}</TD>
```

Comment:
Is there a way to pass in a different `quantity` from a parent component if this is an original SOR? Just thinking it might be neat to surface the logic higher up, and then this component wouldn't need to know/care about whether the SOR is original or not. Just an idea.

Reply:
Good idea - i'll take a look thanks

Reply:
Hey so this is actually coming from 
https://github.com/LBHackney-IT/repairs-hub-frontend/blob/main/src/components/WorkOrder/TasksAndSors/TasksAndSorsTable.js#L33
which is an attribute on the actual object so i'm not setting anything in the parent component for this quantity

Reply:
I suppose we _could_ still manage that by overriding properties on the item from the parent. I kind of like the idea of the row only knowing about one quantity. But this feels really nitpicky and subjective so happy either way.

Not tested:

```
          {tasks.map((entry, index) => (
            <TasksAndSorsRow
              key={index}
              index={index}
              {...entry, quantity: isOriginal ? entry.originalQuantity : entry.quantity }
            />
          ))}
```

Reply:
ooh I like it

Reply:
This syntax doesn't work. I've done some reading and you can't do destructuring and reasssign attributes that easily. Would involve some more steps. Going to just send it down separately

Reply:
Ah :( in that case I'm sorry for the unnecessary detour